#162233562 Write tests for user Authentication

kafuuma opened the pull request:
# What does this PR do?
This PR delivers tests for authentication functionality

# Description of tasks completed
Wrote 4 tests, 3 for user registration and one for user login
# How to run the tests?
coverage run --source='authors/apps/' ./manage.py test 

# Relevant pivotal track stories
[Delivers #162233562]

Commit:
chore(Tests):Write tests for user Authentication

This commit delivers unit tests to test user authentication
functionality.

[Delivers #162233562]

Kasulejoseph commented on authors/apps/authentication/test_authentication.py:
```diff
@@ -0,0 +1,48 @@
+
+
+class TestUsers(APITestCase):
+    client = APIClient()
```
hey, Henry.. it could be better if you defined the test client in a setup method. 
class TestUsers(APITestCase):
    def setUp(self):
        client = APIClient()

joshtrigger replied:
Hey, Henry, the second commit message is not like we had agreed.

kafuuma replied:
alright Josh
I will do better next time, sometimes I forget and go back to default settings

joshtrigger replied:
haha ok

Commit:
made  corrections

kafuuma commented on authors/settings.py:
```diff
@@ -136,7 +136,7 @@
-    'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
-    ),
+    # 'DEFAULT_AUTHENTICATION_CLASSES': (
```
Please Team,
Note that I commented out this part of the code because it was preventing tests from running 
Whoever wrote it should look into it

Kasulejoseph replied:
We should discuss it as a team actually

collin5 replied:
Have you figured why this was preventing tests from passing? @kafuuma

Nta1e replied:
Tests are failing because there is no authentication class. As you see it is authenticating from a class in the `backends.py` file which I created and just made it `pass` for the time being because implementing `JWT`is a feature we will have to work on separately as per the `PT` board

Commit:
chore(Tests):Write tests for user authentication

This commit delivers unit tests to test user authentication
Removed commented code, all tests are passing

[Delivers #162233562]

joshtrigger reviewed (approved):
this is good henry
